build.sbt

```diff
@@ -19,7 +19,7 @@ libraryDependencies ++= Seq(
   filters,
   // Sage packages
   "org.sagebionetworks" % "bridge-base" % "2.7.7",
-  "org.sagebionetworks" % "synapseJavaClient" % "161.0-4-g843de2c",
+  "org.sagebionetworks" % "synapseJavaClient" % "206.0-9-gb27c77a5f7",
```

Comment:
I believe the most up-to-date version is 206.0. The "-9-whatever" seems to be a one-off incremental build.

Reply:
Changed this, works fine